src/main/java/org/datadog/jenkins/plugins/datadog/traces/DatadogTracePipelineLogic.java

```diff
@@ -59,18 +57,20 @@
     private static final String HOSTNAME_NONE = "none";
     private static final Logger logger = Logger.getLogger(DatadogTracePipelineLogic.class.getName());
 
-    private final Tracer tracer;
+    private static final int SIZE_SPANS_SEND_BUFFER = 100;
```

Comment:
How did you decide this number? It seems a bit too small.

Reply:
It was arbitrary. I don't think we're going to have those amounts of spans in a single pipeline, though.
I'm opened to suggestions, but probably a greater number will not affect so much.